content/ernaehrung.md

```diff
@@ -703,3 +703,4 @@ Nummer | Inhalt
 [^58]: (58.) https://veganesk.de/wissen/e-nummern-versteckte-tierprodukte
 [^59]: (59.) https://www.codecheck.info/hintergrund/vegan
 [^60]: (60.) https://eatsmarter.de/ernaehrung/gesund-ernaehren/was-ist-nicht-vegan
+[^61]: (61.) https://www.quarks.de/gesundheit/ernaehrung/darum-ist-agavendicksaft-gar-nicht-so-gut
```

Comment:
Die Quelle wird nicht hinzugefügt, weil wir in diesem Abschnitt keine Aufklärung zum Einfluss der Produkte nehmen. Umwelteinflüsse und Nachhaltigkeit können im Bereich Umwelt geführt werden. Hier soll es in erster Linie nur um die Aufführung ernstzunehmender veganen Alternativen gehen.